Fred2/IO/EnsemblAdapter.py

```diff
@@ -50,7 +50,7 @@ def read_seqs(self, sequence_file):
         recs = sequence_file
         if not isinstance(sequence_file, dict) and not isinstance(sequence_file, list):
             try:
-                with open(sequence_file, 'rb') as f:
+                with open(sequence_file, 'r') as f:
```

Comment:
Bio.StreamModeError: Fasta files must be opened in text mode.
-> changed ‘rb’ (read binary) to ‘r’ (read default - text) eg. [https://biopython.org/wiki/SeqIO 
https://docs.python.org/3.7/library/functions.html#open]